src/components/Common/common.css

```diff
@@ -1,3 +1,46 @@
 p {
-    @apply lil-font-medium lil-text-font-200;
+  font-size: 20px;
+  @apply lil-font-medium lil-text-font-200;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  @apply lil-text-font-500;
+}
+
+.common-padding {
+  @apply lil-py-8;
+}
+
+@screen lg {
+  .full-section {
+    @apply lil-h-screen;
+  }
+  .common-padding {
+    @apply lil-py-24;
+  }
+}
+
+@screen xl {
+  .full-section {
+    @apply lil-h-screen;
+  }
+  .common-padding {
+    @apply lil-py-24;
+  }
+}
```

Comment:
y do we need to define other class for it ❓

Reply:
it seems cleaner than defining padding & height classes for multiple sizes for all the sections separately